homeassistant/components/google_generative_ai_conversation/services.yaml

```diff
@@ -14,3 +14,19 @@ generate_content:
       selector:
         text:
           multiple: true
+generate_image:
```

Comment:
Should this be called generate_media to allow generating e.g. a video? I'd prefer re-using the generate_content though.